component/src/main/java/org/exoplatform/rhmanagement/services/BalanceHistoryService.java

```diff
@@ -108,15 +108,15 @@ public void remove(BalanceHistoryDTO entity) {
           Profile profile = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, entity.getUpdaterId(), false).getProfile();
           entity.setUpdaterId(profile.getFullName());
         } catch (Exception e) {
-          LOG.debug("cannot get profile of"+ entity.getUpdaterId());
+          LOG.error("cannot get profile of"+ entity.getUpdaterId());
         }
       }
       if(entity.getUserId()!=null) {
         try {
           Profile profile = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, entity.getUserId(), false).getProfile();
           entity.setUserId(profile.getFullName());
         } catch (Exception e) {
-          LOG.debug("cannot get profile of"+ entity.getUserId());
+          LOG.error("cannot get profile of"+ entity.getUserId());
         }
       }
       dtos.add(convert(entity));
```

Comment:
Not sure about the consistency of the entity here when an exception is rised in the previous bloc. For example: 
Profile profile = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, entity.getUpdaterId(), false).getProfile();
          entity.setUpdaterId(profile.getFullName());
          entity.setUpdaterId(profile.getFullName());
If an exception would be rised by this bloc, the entity would not have an updatedId despite that it exists. The same comment is relevent to the other blocs

Reply:
`entity.getUserId()!=null` we already check the `UpdaterId`and `UserId`

Reply:
Do not really agree. The exception can be raised by the identityManager.getOrCreateIdentity(..
Anyway, I consider that this is out of the OPS scope.